ortho_seq_code/utils.py

```diff
@@ -11,9 +12,118 @@ def get_seq_info(seqf):
     sites = max(seq_series.str.len())
     pop_size = len(seq_series)
     seq_list = list(np.unique(list("".join(list(seq_series)))))
-    dm = len(seq_list)
     for i in seq_series:
         if i == "\n":
             pop_size -= 1
             seq.remove(i)
-    return [dm, sites, pop_size, seq, seq_series]
+    # Autopadding lowercase n's
+    if len(min(seq, key=len)) != len(max(seq, key=len)):
```

Comment:
Something like`list(seq_series.apply(lambda row: row + (sites - len(row))*"n" + "\n"))` should give back the same results as this block down to line 26. I would probably break it up a bit so it's clearer like,

```
padded_seqs = seq_series.apply(lambda row: row + (sites - len(row))*"n")
seq = [f"{e}\n" for e in padded_seqs]
```